src/LightPanels/LightPanels.less

```diff
@@ -95,6 +95,7 @@
 
 		> .offscreen {
 			visibility: hidden;
+			opacity: 0;
```

Comment:
Hi @yeram, this will not always be the case i.e. if `popOnBack: false` is set and/or we are not caching views.